client/transactions/list/test/index.tsx

```diff
@@ -555,6 +587,7 @@ describe( 'Transactions list', () => {
 				'"Transaction Id"',
 				'"Date / Time"',
 				'Type',
+				'Channel',
```

Comment:
@elazzabi: since we introduced a new column, is it included in CVS export (both client-side and server-side)?

Reply:
Yes. It is covered by the test here, but I made a manual test
<img width="613" alt="Screen Shot 2022-07-19 at 14 49 49" src="https://user-images.githubusercontent.com/12542046/179766760-8ffb848d-4339-4983-9e62-8a77d2ed7be1.png">